system-addon/lib/TopSitesFeed.jsm

```diff
@@ -132,7 +132,7 @@ this.TopSitesFeed = class TopSitesFeed {
    * @param target Optional port/channel to receive the update. If not provided,
    *               the update will be broadcasted.
    */
-  async refresh(target = null) {
+  async refresh(target = null, dispatch = true) {
```

Comment:
Having multiple options here makes calling things clunky. Especially when passing in boolean args. It looks like we don't need to support `target` anymore. And the `dispatch` name is pretty bad name especially because `dispatch` is an existing method vs here it's a condition controlling logic to either `dispatch` when `true` or `dispatch` when `false`.